src/utils/constants.js

```diff
@@ -40,6 +40,7 @@ export const WEBCLIENT = "WEBCLIENT";
  * @type {Object}
  */
 export const REQUEST_PARAMS = {
+    DATASET : 'DATASET',
```

Comment:
I guess you are saying I'm not correct enough in my naming there ;-) 

sure, no problem, I can change the constant name only or both, in the former case there is only one other adjustment, in the later I'd have to manipulate the open_with script as well. It's the one that directly adds the query parameter as "dataset=..." and the constant value has to match that.